### PR TITLE
Update Docker

### PR DESCRIPTION
**Pull request by westim**
Docker recently announced the beta for Software & Hardware 2FA for the Docker Hub site.

https://www.docker.com/blog/designing-docker-hub-2fa/

### CODE REVIEW

File: _data/developer.yml [hunk 1]
```diff
@@ -193,6 +193,10 @@ websites:
     twitter: Docker
     facebook: docker.run
```
Comment (Carlgo11) [inline _data/developer.yml]:
```suggestion
```

File: _data/developer.yml [hunk 2]
```diff
@@ -193,6 +193,10 @@ websites:
     twitter: Docker
```
Comment (Carlgo11) [inline _data/developer.yml]:
```suggestion
```

File: _data/developer.yml [hunk 3]
```diff
@@ -193,6 +193,10 @@ websites:
...
     facebook: docker.run
     img: docker.png
+    tfa:
+      - software
```
Comment (Carlgo11) [inline _data/developer.yml]:
```suggestion
      - totp
```

File: _data/developer.yml [hunk 4]
```diff
@@ -193,6 +193,10 @@ websites:
...
+      - hardware
```
Comment (Carlgo11) [inline _data/developer.yml]: By the looks of it they don't support WebAuthn or any other hardware solutions yet.
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/developer.yml (hunk 4 above)
Comment (kmpoppe) [reply _data/developer.yml]: Resolved by 414add7

File: _data/developer.yml (hunk 3 above)
Comment (kmpoppe) [reply _data/developer.yml]: Resolved by 414add7

File: _data/developer.yml (hunk 2 above)
Comment (kmpoppe) [reply _data/developer.yml]: Resolved by 414add7

File: _data/developer.yml (hunk 1 above)
Comment (kmpoppe) [reply _data/developer.yml]: Resolved by 414add7

**Review by kmpoppe — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**